numpy/typing/__init__.py

```diff
@@ -135,7 +135,7 @@
 Record array dtypes
 ~~~~~~~~~~~~~~~~~~~
 
-The dtype of `numpy.recarray`, and the `numpy.rec` functions in general,
+The dtype of `numpy.recarray`, and the `numpy.record` functions in general,
```

Comment:
FWIW I think this one intended to refer to the `numpy.rec` module, not the `numpy.record` scalar. I'd vote for:

```suggestion
The dtype of `numpy.recarray`, and the ``numpy.rec`` functions in general,
```

Reply:
Will update it in next push.

Reply:
I used :ref:`routines.array-creation.rec` instead since all functions in this part are all functions of numpy.rec module. Just using ``numpy.rec`` may let users confused since there are no ``numpy.rec`` module section.